system-variables.md

```diff
@@ -1067,7 +1136,8 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 
 ### tidb_general_log
 
-- Scope: INSTANCE
+- Scope: GLOBAL
+- Persists to cluster: No
```

Comment:
The documentation for Instance scope variables changes to "GLOBAL + Persists = No". See: https://github.com/pingcap/tidb/blob/master/docs/design/2021-12-08-instance-scope.md